### ISSUE TITLE
[Bug]: A11y addon is not loaded/working in 7.6.0-alpha

### ISSUE THREAD

**Issue by Spielboerg** [storybookjs/storybook#24910]
### Describe the bug

The accessibility tab is not visible with the current alpha version [A11y addon](https://storybook.js.org/addons/@storybook/addon-a11y).

Expected behavior (screenshot from version 7.5.1):
<img width="488" alt="Bildschirmfoto 2023-11-20 um 10 58 14" src="https://github.com/storybookjs/storybook/assets/7893762/cf1a8c0f-8de2-41f9-ac24-9a8cf1c1e463">

Actual behavior:
<img width="555" alt="Bildschirmfoto 2023-11-20 um 10 56 43" src="https://github.com/storybookjs/storybook/assets/7893762/cb89af00-8493-47ba-8875-b08ace06056a">

### To Reproduce

https://stackblitz.com/edit/github-ga8mcr?file=package.json,.storybook%2Fmain.js

Note, that `@storybook/addon-a11y` is installed and the addon is added in the `main.js`.

**Comment by jonniebigodes** [comment]
@Spielboerg, thanks for taking the time to report this. We appreciate it 🙏 ! I've run the addon in the latest stable version (e.g., 7.6), and it seems ok. The addon is loading as it should, as you can see it deployed [here](https://656f640d39eb488a7d99c06e-lvznizseys.chromatic.com/?path=/story/example-button--primary). 

I did some additional testing, and it seems that the issue is solved even with the latest alpha version, as you can see deployed [here](https://stackblitz.com/edit/github-4izpdm?file=package.json) on StackBlitz.

Can you check on your end to see if the issue is still present?

Looking forward to hearing from you so we can continue working on this issue or close it.

Have a great day.

Stay safe

**Comment by Marklb** [comment]
@Spielboerg Your reproduction does not have `@storybook/addon-a11y` installed. This screenshot is from that Stackblitz, showing the warning and the dependencies that are missing it: 
![image](https://github.com/storybookjs/storybook/assets/8348647/6562cb30-c176-45d1-92dd-a53a1dbe48b7)

After running `npm i -D @storybook/addon-a11y@7.6.0-alpha.4`, the addon tab displayed.
